logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/mlsmigration/MLSMigrationWorker.kt

```diff
@@ -42,7 +42,7 @@ internal class MLSMigrationWorkerImpl(
     override suspend fun runMigration() =
         syncMigrationConfigurations().flatMap {
             userConfigRepository.getMigrationConfiguration().getOrNull()?.let { configuration ->
-                if (configuration.status.toBoolean() && configuration.hasMigrationStarted()) {
```

Comment:
FYI: this was extra check, since we check the status in the `.hasMigrationStarted` function.